app/connectors/des/DesConnector.scala

```diff
@@ -32,14 +32,14 @@ import scala.concurrent.{ExecutionContext, Future}
 @Singleton
 class DesConnector @Inject() (servicesConfig: ServicesConfig, httpClient: HttpClient, configuration: Configuration)(implicit ec: ExecutionContext) {
 
-  val serviceURL: String = servicesConfig.baseUrl("des")
-  val authorisationToken: String = configuration.get[String]("microservice.services.des.authorization-token")
-  val serviceEnvironment: String = configuration.get[String]("microservice.services.des.environment")
-  val obligationsUrl: String = configuration.get[String]("microservice.services.des.obligations-url")
-  val financialsUrl: String = configuration.get[String]("microservice.services.des.financials-url")
-  val customerUrl: String = configuration.get[String]("microservice.services.des.customer-url")
+  private val serviceURL: String = servicesConfig.baseUrl("des")
+  private val authorisationToken: String = configuration.get[String]("microservice.services.des.authorization-token")
```

Comment:
I guess that these URLs (or paths actually) don't change from environment to environment. Can we hardcode them as strings here in future? It will be easier to reason about the code then :)

Reply:
I put them in config as not 100% sure of the paths